src/Uploader/useUploader.ts

```diff
@@ -77,11 +77,31 @@ export default function useUploader({
     const bytesToMb = maxFileSize / 1024 / 1024;
     const limitSize = `${bytesToMb.toFixed()} MB`;
 
+    // check if the uploader is for images
+    const isImageUploader = Object.keys(acceptedFiles ?? {}).some(key =>
+      key.includes("image/")
+    );
+
     // set error message based on error code
-    const errorMessage =
-      errorCode === "file-invalid-type"
-        ? "File type must be a .gif, .jpg, .jpeg, .png, .webp"
-        : `File size exceeds the limit of ${limitSize}`;
+    let errorMessage = "";
+    switch (errorCode) {
+      case "file-invalid-type":
+        errorMessage = isImageUploader
+          ? "File type must be a .gif, .jpg, .jpeg, .png, .webp"
```

Comment:
Are these file extensions fixed by the component? Always feels a bit dangerous to be hardcoding things like this.

Reply:
Yes, these extensions are fixed for image uploader. We dont have have a `acceptedFiles` prop for this, that's why i hardcoded this.

Reply:
react-dropzone already gives you this error message based on the `acceptedFiles` property. If the built in error message is not sufficient, then I suggest you build it from the `acceptedFiles` property here as well instead hard coding ImageUploader logic into a hook that doesn't know anything about the type of uploader it is used in.